library/accessory_workflows/build_indices/commands.sh

```diff
@@ -0,0 +1,2 @@
+#MOUSE
+cromwell-tools submit --url "https://cromwell.caas-prod.broadinstitute.org" --collection-name "pipeline-surge" --service-account-key /Users/kkonwar/.ssh/pipelines_surge_credentials.json --wdl BuildIndices.wdl -i mouse_inputs.json -l mouse-build-label.json 
```

Comment:
Should this be checked in? No one else can use it without your service account key

Reply:
removed